server/audit_math/minerva.py

```diff
@@ -1,11 +1,111 @@
+# pylint: disable=invalid-name
 """
-An implementation of Minerva
+Library for performing a Minerva-style ballot polling risk-limiting audit,
+as described by Zagórski et al https://arxiv.org/abs/2008.02315
+
+Note that this library works for one contest at a time, as if each contest being
+targeted is being audited completely independently.
+
+TODO: ensure the no-losers case is handled
+TODO: if necessary pull out risks for individual contests
+
 """
-# TODO: remove the following once we are using the arguments
-# pylint: disable=unused-argument
-from typing import Dict, Tuple, Optional
+from decimal import Decimal
+from collections import defaultdict
+import logging
+from typing import List, Dict, Tuple, Optional
 
+from athena.audit import Audit  # type: ignore
 from .sampler_contest import Contest
+from .shim import minerva_sample_sizes  # type: ignore
+
+# FIXME: make this an environmental variable
+MINERVA_MULTIPLE = 1.5
+
+
+def compute_cumulative_sample(sample_results):
```

Comment:
I'm wondering if I should pull this up to the ballot_polling module so we don't have to repeat the code...

Reply:
This is actually being bypassed right now, and I actually may not need it at all in Minerva, when I redo this code.

Reply:
I pulled it out.